doc/website/docusaurus.config.ts

```diff
@@ -4,7 +4,7 @@ import type * as Preset from '@docusaurus/preset-classic';
 
 const config: Config = {
   title: 'Holos',
-  tagline: 'Holos makes it easier for platform teams to integrate software into their platform.',
+  tagline: 'An easier way for platform teams to integrate software into their platform',
```

Comment:
* The duplicate "Holos Holos" on the landing page felt redundant and made it a bit clunky to read.
* The final "into their platform" feels redundant but might be important to convey that we're integrating INTO a platform. I'm also fine with cutting everything after "software".

Reply:
Maybe "An easier way for platform teams to integrate software components."